.buildkite/scripts/trigger_integrations_in_parallel.sh

```diff
@@ -52,6 +52,7 @@ for package in ${PACKAGE_LIST}; do
       artifact_paths:
         - build/test-results/*.xml
         - build/benchmark-results/*.xml
+        - build/test-coverage/*.out
```

Comment:
Files generated by elastic-package are with `xml` extension

```suggestion
        - build/test-coverage/*.xml
```